docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -129,13 +139,25 @@ For an end user, the UX to update the kubernetes version of the control plane an
 As an end user, I want to be able to easily scale up/down the number of replicas for each set of worker nodes in the cluster.
 - Currently, (for a cluster with 3 machine deployments) this is possible by updating these three different objects representing the sets of worker nodes in the pool. An easier user experience would be to update a single object to enable the scaling of multiple sets of worker nodes.
 
+#### Story 4 - Ability to customize individual clusters
+As a Cluster operator, I want to customize individual clusters simply by providing variables in the Cluster resource.
+
+**Note**: Without this ability all Clusters of the same ClusterClass would be the same apart from a few properties like Kubernetes version, 
+labels and annotations. This would make it necessary to create new ClusterClasses for small deviations and thus limit the number of variants 
+a single ClusterClass could address.
+
+#### Story 5 - Provide powerful and flexible options to define ClusterClass customizations
```

Comment:
Would it make sense to add a user story about variable mutations? What happens if I change the value of a variable after clusters using it are created? Are existing clusters updated? If so, in what manner? What about immutable fields?

Reply:
Added stories for basic mutability (without patches/variables) and one for variable mutation. I also updated the update cluster behavior section to bring it inline with our current implementation (and also to clarify that Cluster mutability is not introduced by patches, we just also leverage it there)